wlauto/result_processors/cpustate.py

```diff
@@ -142,9 +149,13 @@ def set_initial_state(self, context):
         self.logger.debug('Writing initial frequencies into trace...')
         device = context.device
         cluster_freqs = {}
+        cluster_max_freqs = {}
+        del self.max_freq_list[:]
```

Comment:
I did this originally, but pylint gave me this:
`W:153, 8: Attribute 'max_freq_list' defined outside __init__ (attribute-defined-outside-init)`
As such i changed it to del instead.
If you prefer this behaviour, i will add:
`# pylint: disable=attribute-defined-outside-init`